bonsai/Modules/Home/View/Budget/BudgetView.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  BudgetView.swift
+//  bonsai
+//
+//  Created by Максим Алексеев  on 07.12.2021.
+//
+
+import SwiftUI
+
+struct BudgetView: View {
+    var body: some View {
+        VStack(alignment: .leading) {
+            HStack {
```

Comment:
I think its might be better make HStack in separate view to reduce this long line code BudgetView